assets/performanceprofile/tuned/openshift-node-performance-amd-x86

```diff
@@ -0,0 +1,25 @@
+[main]
+summary=Platform specific tuning for AMD x86
+
+[bootloader]
+cmdline_iommu_amd=amd_iommu=on iommu=pt
+
+{{if .PerPodPowerManagement}}
+cmdline_pstate=amd_pstate=active
+{{else if .HardwareTuning}}
+cmdline_pstate=amd_pstate=passive
+{{else}}
+cmdline_pstate=amd_pstate=guided
+{{end}}
```

Comment:
Hmm... how did you decide on these? I would have thought we would use active by default and then passive only when .PerPodPowerManagement was enabled.

Reply:
I based these on the intel ones but apparently after I mixed them up, so I will update these too